lucien-ios/Extensions/UILabel+Extensions.swift

```diff
@@ -18,4 +18,12 @@ extension UILabel {
             attributedText = attributedString
         }
     }
+
+    func setLineHeight(text: String, lineSpacing: Double) {
+        let attributedString = NSMutableAttributedString(string: text)
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.lineSpacing = CGFloat(lineSpacing)
+        attributedString.addAttribute(NSAttributedStringKey.paragraphStyle, value:paragraphStyle, range:NSMakeRange(0, attributedString.length))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Legacy Constructor Violation: Swift constructors are preferred over legacy convenience functions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Alegacy_constructor)

Reply:
You need to fix warnings from this